core/src/main/java/org/frankframework/scheduler/job/CheckReloadJob.java

```diff
@@ -84,15 +84,17 @@ public void execute() {
 					configNames.add(configName);
 					if (DATABASE_CLASSLOADER.equals(configuration.getClassLoaderType())) {
 						stmt.setString(1, configName);
+						stmt.setString(2, booleanValueTrue);
+						stmt.setString(3, booleanValueTrue);
```

Comment:
Hier ook toch echt even opletten, het veld in de database is een [boolean](https://github.com/frankframework/frankframework/blob/master/core/src/main/resources/IAF_Util/IAF_DatabaseChangelog.xml#L107), het ligt aan de jdbc driver of die dan de conversie (uberhaupt) doet van string naar 0/1, true/false of T/F (afhankelijk van de database).

Reply:
Of
```
stmt.setBoolean(2, true);
stmt.setBoolean(3, true);
```

Reply:
En die conversie gaat misschien anders als je die als parameter met de `setString()` set? Oke, dan zet ik dit weer terug.